Add k-NN radial search doc (#7050)
opensearch-project/documentation-website: _search-plugins/knn/approximate-knn.md

`k` is the number of neighbors the search of each graph will return. You must also include the `size` option, which
indicates how many results the query actually returns. The plugin returns `k` amount of results for each shard
(and each segment) and `size` amount of results for the entire query. The plugin supports a maximum `k` value of 10,000.

> Starting from OpenSearch version 2.14.0, in addition to use `k`, `min_score` and `max_distance` can be optionally utilized for [radial search]({{site.url}}{{site.baseurl}}/search-plugins/knn/radial-search-knn/).

Author: Not sure if the comma position need adjusted:

in addition to use the `k` variables, `min_score` and `max_distance` can be optionally utilized for [radial search]({{site.url}}{{site.baseurl}}/search-plugins/knn/radial-search-knn/).

`k` is the number of neighbors the search of each graph will return. You must also include the `size` option, which
indicates how many results the query actually returns. The plugin returns `k` amount of results for each shard
(and each segment) and `size` amount of results for the entire query. The plugin supports a maximum `k` value of 10,000.

> Starting in OpenSearch 2.14, in addition to use the `k` variable, `min_score` and `max_distance` can be optionally utilized for [radial search]({{site.url}}{{site.baseurl}}/search-plugins/knn/radial-search-knn/).

Reviewer: ```suggestion
Starting in OpenSearch 2.14, in addition to use the `k` variable,  both the `min_score` and `max_distance` variables can be used for [radial search]({{site.url}}{{site.baseurl}}/search-plugins/knn/radial-search-knn/).
```

> Starting in OpenSearch 2.14, in addition to use the `k` variable,  both the `min_score` and `max_distance` variables can be used for [radial search]({{site.url}}{{site.baseurl}}/search-plugins/knn/radial-search-knn/).

Reviewer 2: ```suggestion
Starting in OpenSearch 2.14, in addition to using the `k` variable, both the `min_score` and `max_distance` variables can be used for [radial search]({{site.url}}{{site.baseurl}}/search-plugins/knn/radial-search-knn/).
```

To learn about using k-NN search with nested fields, see [k-NN search with nested fields]({{site.url}}{{site.baseurl}}/search-plugins/knn/nested-search-knn/).

### Using approximate radial search

> To learn more about radial search feature, see [k-NN radial search]({{site.url}}{{site.baseurl}}/search-plugins/knn/radial-search-knn/).

Reviewer 2: ```suggestion
To learn more about the radial search feature, see [k-NN radial search]({{site.url}}{{site.baseurl}}/search-plugins/knn/radial-search-knn/).
```
